app/src/main/java/com/github/polybooks/core/database/implementation/SaleDatabase.kt

```diff
@@ -1,33 +1,27 @@
 package com.github.polybooks.core.database.implementation
 
-import android.util.Log
 import com.github.polybooks.core.*
 import com.github.polybooks.core.database.DatabaseException
 import com.github.polybooks.core.database.LocalUserException
+import com.github.polybooks.core.database.interfaces.*
 import com.github.polybooks.core.database.interfaces.SaleDatabase
-import com.github.polybooks.core.database.interfaces.SaleSettings
-import com.github.polybooks.core.database.interfaces.SaleOrdering
-import com.github.polybooks.core.database.interfaces.SaleQuery
-
-import com.google.firebase.firestore.CollectionReference
-import com.google.firebase.firestore.FirebaseFirestore
-import com.google.firebase.firestore.Query
-
+import com.github.polybooks.core.database.interfaces.SaleOrdering.*
+import com.github.polybooks.utils.url2json
 import com.google.android.gms.tasks.Task
 import com.google.firebase.Timestamp
 import com.google.firebase.firestore.*
-
+import com.google.firebase.firestore.Query
 import java.util.concurrent.CompletableFuture
-import kotlin.collections.HashMap
 
-class SaleDatabase : SaleDatabase {
+private const val COLLECTION_NAME = "sale2"
```

Comment:
This private variable `COLLECTION_NAME` is outside the SaleDatabse class, is it intended? If yes, why using the private keyword?

Reply:
It is a static constant. So I define it as a top level declaration.